### DISCUSSION TITLE
Installation error in elementary os 6(Unable to install goocanvas-3)

### DISCUSSION THREAD

**Discussion by DuttaAB-dev** [akiraux/Akira#610] [Q&A] [answered]
Hi! I am unable to build Akira from source as goocanvas 3 is not available in the ubuntu 20.04 repos and building it from source is not working for me as there are no clear instructions. Can anyone help me out on this?

**Comment by Suzie97** [comment, marked as answer]
I just installed goocanvas for compiling Akira on Odin yesterday.
 https://github.com/GNOME/goocanvas/tree/goocanvas-3.0-meson.

Make sure to use the meson branch.

Run 
```sh
meson build --prefix=/usr
cd build
sudo ninja install
```
PS Thanks to @Abdallah-Moh for guiding me on how to do it.

**Reply by ghost** [reply]
Make sure to install all packages that goocanvas will require too

**Reply by DuttaAB-dev** [reply]
> I just installed goocanvas for compiling Akira on Odin yesterday.
> https://github.com/GNOME/goocanvas/tree/goocanvas-3.0-meson.
> 
> Make sure to use the meson branch.
> 
> Run
> 
> ```shell
> meson build --prefix=/usr
> cd build
> sudo ninja install
> ```
> 
> PS Thanks to @Abdallah-Moh for guiding me on how to do it.

Thanks, it worked!

**Reply by ghost** [reply]
@DuttaAB-dev if it worked you can mark this as answered
